### PR TITLE
Doc - code style written down on example

### PR DESCRIPTION
**Pull request by rfree-d**

### CODE REVIEW

File: src/style_example.cpp [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		int m_age; ///< years since birth
+		static long int s_count_all; ///< count of all humans (now alive)
+};
+long int c_human::s_count_all; ///< count of all humans (now alive)
+
+
+// ==================================================================
+
+int foo(int b) {
+	int a=42;
```
Comment (mikurys) [inline src/style_example.cpp]: maybe will be better if we add spaces around operator=

File: src/style_example.cpp [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	if (a>b) {
+		return a+b;
+	}
+	if (a<b) return a+b*2;
```
Comment (mikurys) [inline src/style_example.cpp]: also will look better if we add spaces around logical and arithmetic operators

File: src/style_example.cpp [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+free function:          free_function()
+
+member variable:        m_member_variable
+member variable in t_*: member_variable // e.g.: my_point.x=5
+local variable:         some_variable
+function parameter:     some_parameter
+function parameter:     _some_parameter // to avoid duplication when needed
+function parameter:     arg_color or opt_color // sometimes to be more clear
+static variable:        s_static_variable
+global variable:        g_global_variable // use very rarerly!
```
Comment (mikurys) [inline src/style_example.cpp]: typo rarely

Comment (rfree-d) [reply src/style_example.cpp]: 🆗

File: src/style_example.cpp (hunk 1 above)
Comment (rfree-d) [reply src/style_example.cpp]: 🆗 explained it depends, and we recommend compressed for short vars, and with space for long

File: src/style_example.cpp (hunk 2 above)
Comment (rfree-d) [reply src/style_example.cpp]: as above

**Pull request merged**